app/controllers/admin/feedbacks_controller.rb

```diff
@@ -32,8 +32,8 @@ def permitted_kinds
   def matching_feedbacks
     feedbacks = Feedback
     if params[:search_kind].present? && permitted_kinds.include?(params[:search_kind])
```

Comment:
`permitted_kinds` is a method that lists the permitted kind scopes that can be called on the Feedback model.

This line verifies that the method is whitelisted (that the `permitted_kind` includes it)

The value of `@search_kind` is a scope that is called directly on `Feedback` (not an array, not nil).